packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -80,35 +80,35 @@ const Options = styled.ul`
   list-style-type: none;
 
   & li + li {
-    padding-top: ${GEL_SPACING};
+    margin-top: 0.75rem;
+    float: ${ltrRtl('right', 'left')};
   }
 
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    width: 18%;
-    float: ${ltrRtl('right', 'left')};
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    & li + li {
+      margin-top: 1rem;
```

Comment:
Where does this come from on the design? Can that use a GEL constant?

Reply:
Good spot, yeah i'll do that now